### PR TITLE
feat: support auto add label when pr created

### PR DESCRIPTION
**Pull request by littlejiancc**

### CODE REVIEW

File: gh/gh.go [hunk 1]
```diff
@@ -66,3 +66,20 @@ func GetRepoStar(owner, repo string) (int, error) {
 	}
 	return r.GetStargazersCount(), nil
 }
+
+func ListLabel(owner string, repo string, page, perPage int) ([]*github.Label, error) {
```
Comment (sfwn) [inline gh/gh.go]: ListLabels

File: handlers/issue/title/label/handler.go [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+	`github.com/erda-project/erda-bot/gh`
+	`github.com/erda-project/erda-bot/handlers`
+	`github.com/erda-project/erda-bot/handlers/issue/comment`
+)
+
+var semanticLabelMap = map[string]string{
+	"feat":     "feature",
+	"fix":      "bugfix",
+	"refactor": "refactor",
+	"docs":     "",
```
Comment (sfwn) [inline handlers/issue/title/label/handler.go]: fulfill these semantic labels please

File: handlers/issue/title/label/handler.go [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+	}
+	for _, v := range labels {
+		if _, ok := labelsExistMap[v]; !ok {
+			labelsNew = append(labelsNew, v)
+		}
+	}
+	return labelsNew, nil
+}
+
+func findEffectLabels(e github.IssueCommentEvent, labels []string) ([]string, error) {
```
Comment (sfwn) [inline handlers/issue/title/label/handler.go]: strange name. please add comment and ut for this function.

File: handlers/issue/title/label/handler.go [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+	e := req.Event.(github.IssueCommentEvent)
+	labelsNew, err := findNewLabels(e)
+	if err != nil {
+		logrus.Errorf("failed to findNewLabels, err: %v", err)
+		return
+	}
+	if len(labelsNew) == 0 {
+		h.DoNexts(ctx, req)
+	}
+	labelsEffect, err := findEffectLabels(e, labelsNew)
```
Comment (sfwn) [inline handlers/issue/title/label/handler.go]: no need find twice.
just find required labels and add idempotently.

**Review by sfwn — CHANGES_REQUESTED** [review]

File: gh/gh.go (hunk 1 above)
Comment (littlejiancc) [reply gh/gh.go]: done

File: handlers/issue/title/label/handler.go (hunk 3 above)
Comment (littlejiancc) [reply handlers/issue/title/label/handler.go]: done

File: handlers/issue/title/label/handler.go (hunk 4 above)
Comment (littlejiancc) [reply handlers/issue/title/label/handler.go]: done

File: handlers/issue/title/label/handler.go (hunk 2 above)
Comment (littlejiancc) [reply handlers/issue/title/label/handler.go]: just add a few

**Review by sfwn — APPROVED** [review]

**Pull request merged**